wordpress-mobile/WordPress-Android
# Remove comments list feature flag.

This is a cleanup PR that removes the `UnifiedCommentsListFeatureConfig` feature flag and aligns the places where it was used.

## To test
- Check CI is green
- Smoke test the comments list (both in My SIte and Notifications. Check single and batch moderation, pagination, PTR, comment details and its behaviours. Check different filtered lists...) checking no regression was introduced.

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
No unintended area of impact should be there; anyways we are relying on already available unit testing and manual testing.

3. What automated tests I added (or what prevented me from doing so)
This is removing unused feature flag so no tests addition needed.

PR submission checklist:
